astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -224,183 +219,20 @@ def execute_complete(self, context: Context, event: dict[str, str | list[str]] |
             raise AirflowException("Did not receive valid event from the trigerrer")
 
 
-class SnowflakeSqlApiOperatorAsync(SnowflakeOperator):
+class SnowflakeSqlApiOperatorAsync(SnowflakeSqlApiOperator):
+    """
+    This class is deprecated and will be removed in 2.0.0.
```

Comment:
Why will we remove this in 2.0.0? I think the operators and sensors will keep proxying it right? Only the hooks & triggers would be removed, right?

Reply:
yes you are correct, thanks, fixed it